core/domain/exp_domain.py

```diff
@@ -2811,6 +2792,48 @@ def _convert_states_v12_dict_to_v13_dict(cls, states_dict):
                 state_dict['interaction']['solution'] = None
         return states_dict
 
+    @classmethod
+    def _convert_states_v13_dict_to_v14_dict(cls, states_dict):
+        """Converts from version 13 to 14. Version 14 adds
+        audio translations to feedback, hints, and solutions.
+        """
+        for state_dict in states_dict.values():
+            if (state_dict['interaction']['default_outcome'] is not None and
+                    len(state_dict['interaction']['default_outcome']
+                        ['feedback'])) > 0:
+                default_outcome_feedback = (
+                    state_dict['interaction']['default_outcome']['feedback'][0])
+            else:
+                default_outcome_feedback = ''
+            if state_dict['interaction']['default_outcome'] is not None:
+                state_dict['interaction']['default_outcome']['feedback'] = {
+                    'html': default_outcome_feedback,
+                    'audio_translations': {}
+                }
+            for answer_group_dict in state_dict['interaction']['answer_groups']:
+                if len(answer_group_dict['outcome']['feedback']) > 0:
+                    feedback = answer_group_dict['outcome']['feedback'][0]
+                else:
+                    feedback = ''
+                answer_group_dict['outcome']['feedback'] = {
+                    'html': feedback,
+                    'audio_translations': {}
+                }
+            for hint_dict in state_dict['interaction']['hints']:
```

Comment:
Cf. previous point -- consider making state_dict['interaction']['hints'] just a list of SubtitledHtml objects? Unless we want to add stuff to them in the future, in which case it might be worth just renaming hint_text to hint_content for parity with the usual "state content".

Reply:
changed to hint_content